src/clr-angular/data/datagrid/datagrid-column-toggle.spec.ts

```diff
@@ -82,6 +83,29 @@ export default function(): void {
         expect(context.clarityElement.querySelectorAll('.column-switch').length).toBe(0);
       });
 
+      it('#anchor should have aria-controls set', function() {
+        expect(
+          context.clarityElement.querySelector('button.column-toggle--action').attributes['aria-controls'].value
+        ).toBeDefined();
```

Comment:
I think this can be defined but still empy, we should check it has a value. Honestly, this seems unnecessary as you are checking more specifically in the next test.

Reply:
You are right, this test doesn't prove anything :) - will remove it